src/main/java/com/almightyalpaca/discord/jdabutler/Bot.java

```diff
@@ -27,115 +16,126 @@
-		final int proxyPort = Bot.config.getInt("proxy.port", 8080);
-		final boolean useProxy = Bot.config.getBoolean("proxy.use", false);
-		if (useProxy) {
-			builder.setProxy(new HttpHost(proxyAdress, proxyPort));
-		}
+        final String proxyAdress = Bot.config.getString("proxy.host", "");
+        final int proxyPort = Bot.config.getInt("proxy.port", 8080);
+        final boolean useProxy = Bot.config.getBoolean("proxy.use", false);
+        if (useProxy) {
+            builder.setProxy(new HttpHost(proxyAdress, proxyPort));
+        }
 
-		Bot.config.save();
-		Bot.listener = new EventListener();
-		builder.addListener(Bot.listener);
+        Bot.config.save();
+        Bot.listener = new EventListener();
+        builder.addListener(Bot.listener);
+        builder.addListener((dispatcher = new Dispatcher()));
 
-		Bot.jda = (JDAImpl) builder.buildBlocking();
+        Bot.jda = (JDAImpl) builder.buildBlocking();
 
-		SimpleLog.addListener(new LogListener() {
+        SimpleLog.addListener(new LogListener() {
 
-			@Override
-			public void onError(final SimpleLog log, final Throwable t) {
-				log.log(Level.FATAL, ExceptionUtils.getStackTrace(t));
-			}
+            @Override
+            public void onError(final SimpleLog log, final Throwable t) {
+                log.log(Level.FATAL, ExceptionUtils.getStackTrace(t));
+            }
 
-			@Override
-			public void onLog(final SimpleLog log, final Level level, final Object message) {
-				if (level.getPriority() >= Level.INFO.getPriority()) {
-					String format = "`" + Bot.LOGFORMAT.replace("%time%", Bot.DATEFORMAT.format(new Date())).replace("%level%", level.getTag()).replace("%name%", log.name).replace("%text%", String
-							.valueOf(message)) + "`";
-					if (format.length() >= 2000) {
-						format = format.substring(0, 1999);
-					}
-					Bot.getChannelLogs().sendMessage(format).queue();
-				}
-			}
-		});
+            @Override
+            public void onLog(final SimpleLog log, final Level level, final Object message) {
+                if (level.getPriority() >= Level.INFO.getPriority()) {
+                    String format = "`" + Bot.LOGFORMAT.replace("%time%", Bot.DATEFORMAT.format(new Date())).replace("%level%", level.getTag()).replace("%name%", log.name).replace("%text%", String
+                            .valueOf(message)) + "`";
+                    if (format.length() >= 2000) {
+                        format = format.substring(0, 1999);
+                    }
+                    if (Bot.getChannelLogs() != null)
+                        Bot.getChannelLogs().sendMessage(format).queue();
+                }
+            }
+        });
 
-		EventListener.start();
+        EventListener.start();
 
-		Bot.jda.getPresence().setGame(Game.of("JDA"));
+        Bot.jda.getPresence().setGame(Game.of("JDA"));
 
-	}
+    }
 
-	public static void shutdown() {
-		Bot.jda.shutdown();
-	}
+    public static void shutdown() {
+        Bot.jda.shutdown();

```

Comment:
So, is this PR meant to change the indentation style of the project or add a help command?